pennotools/src/qrekening/process.py

```diff
@@ -0,0 +1,151 @@
+import datetime
+from .qPerson import DavilexPerson
+from .config import sepa_headers, output_headers
+from members.models import Person
+
+
+def read_exc(wb, debet, persons):
+    """Read the given excel sheet.
+
+    - read_exc: reads an excel sheet
+    - Input: | sheet: a string of the path to the excel sheet
+             | debet: boolean whether it is debet (true) or credit (false)
+             | persons: dict (qID:DavilexPerson)
+    - Returns: list(DavilexPerson)
+    """
+    for s in wb.sheets():
+        col_names = s.row(0)
+        person_finished = True
+        for row in range(1, s.nrows):
+            col_value = {}
+            for name, col in zip(col_names, range(s.ncols)):
+                value = s.cell(row, col).value
+                col_value[name.value] = value
+            if col_value['Zoekcode'] != '' and 'totaal' in col_value['Omschrijving']:
+                person_finished = True
+            elif not person_finished:
+                current_person.addBoekstuk(col_value, debet)
```

Comment:
`add_boekstuk`? Also in other places. Also, `current_person` is undefined.

Could you add a unittest or include sample files so that I can add tests.